02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -42,6 +42,8 @@ Essentially, there are 3 factors that affect whether documentation is "good" or
 2. Its type: the question being answered, targeting a specific subsection of the audience
 3. How accurately it reflects the desired version of the code/project
 
+It's important to note that this section is sourced primarily from two blog posts, [What Nobody Tells You About Documentation](https://www.divio.com/blog/documentation/) and [Teach, Don't Tell](http://stevelosh.com/blog/2013/09/teach-dont-tell/) -- a relatively small number of sources, without even considering the topic of source qualifications. This makes the correctness of this section less certain than others in this document, which makes it more likely to change in significant ways as more information is gathered. Therefore, it would be prudent to define a more "flexible" shape than what's described here when including this in the desired outcomes.
```

Comment:
This is really wordy...

How would you render this if you were talking to a 6-year-old? You have these main ideas in your rendering above:
- conclusions come from only 2 sources, not more
- take the best from them, but realize there might be better ideas/practices that they don't mention

Reply:
Ah, that's a great way of phrasing it! I've updated it.